integrationkey/queries.sql

```diff
@@ -48,6 +48,27 @@ WHERE
     id = $1
 FOR UPDATE;
 
+-- name: IntKeyLog :exec
+INSERT INTO uik_logs (
+    integration_key_id,
+    content_type,
+    user_agent,
+    raw_body,
+    status,
+    error_message,
+    received_at
+) VALUES (
+    $1, $2, $3, $4, $5, $6, $7
+)
+ON CONFLICT (integration_key_id) DO UPDATE
+SET
+    content_type = EXCLUDED.content_type,
+    user_agent = EXCLUDED.user_agent,
+    raw_body = EXCLUDED.raw_body,
+    status = EXCLUDED.status,
+    error_message = EXCLUDED.error_message,
+    received_at = EXCLUDED.received_at;
```

Comment:
For testing/consistency, we'll want to use DB time -- if we pass in the `received_at` time, we'll need to do a separate query to get the current time. Instead, we can just rely on the default of `now()` and on the conflict update set the value to the current DB time.

```suggestion
-- name: IntKeyLog :exec
INSERT INTO uik_logs (
    integration_key_id,
    content_type,
    user_agent,
    raw_body,
    status,
    error_message
) VALUES (
    $1, $2, $3, $4, $5, $6
)
ON CONFLICT (integration_key_id) DO UPDATE
SET
    content_type = EXCLUDED.content_type,
    user_agent = EXCLUDED.user_agent,
    raw_body = EXCLUDED.raw_body,
    status = EXCLUDED.status,
    error_message = EXCLUDED.error_message,
    received_at = now();
```